Automattic/woocommerce-payments
# Initiate transactions export in server via download button

Fixes #1484 

#### Changes proposed in this Pull Request

Trigger request to server when requesting a download of transaction table while using date or type filters. Otherwise, only download transactions presented in the table view.

#### Testing instructions

Note this PR relies on this server PR: 1178-gh-Automattic/woocommerce-payments-server

- Via WP-Admin go to **Payments > Transactions**
- Click the download link: https://cloudup.com/ccse5g7T_wc
- Ensure a CSV containing the transactions listed in the table view immediately downloads
- Select any filter (i.e. before December 1)
- Ensure a notice appears letting you know an export will be emailed to you
- Confirm you receive an email containing a download link for an export containing transactions that meet the criteria
- Repeat steps 3-6 with different combinations of filters

-------------------


**Post merge**
